screen-capture-api/README.md

```diff
@@ -0,0 +1,5 @@
+Examples:
+
+- [Screen Capture API example](https://mdn.github.io/dom-examples/screen-capture-api/basic-screen-capture): A basic example showing how to use the [Screen Capture API](https://developer.mozilla.org/docs/Web/API/Screen_Capture_API) to capture a surface (such as a window or tab) and broascast the resulting stream in a `<video>` element.
+- [Element Capture API example](https://mdn.github.io/dom-examples/screen-capture-api/element-capture): The same example, but this time using the [Element Capture API](https://developer.mozilla.org/docs/Web/API/Screen_Capture_extensions/Element_Region_Capture#the_element_capture_api) to restrict the broadcast stream to one particular rendered element inside the same tab the app is running in.
+- [Region Capture API example](https://mdn.github.io/dom-examples/screen-capture-api/region-capture): The same example, but this time using the [Region Capture API](https://developer.mozilla.org/docs/Web/API/Screen_Capture_extensions/Element_Region_Capture#the_region_capture_api) to crop the broadcast stream to the screen area in which one particular element is rendered, inside the same tab the app is running in.
```

Comment:
Both the region and element examples don't seem too different to me. I can see that the element has an outline / border versus the region (tab), but the actual content broadcast doesn't look different. Trying this on Chrome `Version 131.0.6778.86 (Official Build) (arm64)`

Reply:
The examples are identical, apart from using a different API / slightly different code to achieve the same thing.

Chrome automatically draws a blue border around the element being captured.

Reply:
Oh I see, we're using a different API only. Can we add that to the demo `<p>`s?